backend/src/main/java/gov/cdc/usds/simplereport/api/pxp/PatientExperienceController.java

```diff
@@ -137,6 +139,31 @@ public String getObfuscatedPatientNameFromLink(
     return p.getFirstName() + " " + p.getLastName().charAt(0) + ".";
   }
 
+  /**
+   * Given a patient link ID, returns a minimum of information about the patient (obfuscated name)
+   * and facility (name and phone number) that may aid the end user in identify verification.
+   *
+   * <p>This endpoint is unauthorized and therefore does not return fully identifying data.
+   */
+  @GetMapping("/entity")
+  public PxpTestResultUnauthenticatedResponse getTestResultUnauthenticated(
+      @RequestParam("patientLink") UUID patientLink, HttpServletRequest request)
+      throws ExpiredPatientLinkException {
+    var link = _pls.getPatientLink(patientLink);
+
+    if (link.isExpired()) {
+      throw new ExpiredPatientLinkException();
```

Comment:
This might be out of scope for this ticket, but could we update the expiration error to also include patient and facility info? I believe right now it just says "This link is expired, please contact your testing facility for a new one" or similar; it'd be helpful if that error message included the facility info at least.